sky/skylet/events.py

```diff
@@ -157,30 +166,20 @@ def _stop_cluster(self, autostop_config):
                           write_ray_up_script_with_patched_launch_hash_fn(
                               self._ray_yaml_path,
                               ray_up_kwargs={'restart_only': True}))
-                subprocess.run(
-                    [sys.executable, script],
-                    check=True,
-                    # Use environment variables to disable the ray usage
-                    # collection (to avoid overheads and potential issues with
-                    # the usage) as sdk does not take the argument for
-                    # disabling the usage collection.
-                    #
-                    # Do not inherit from os.environ, which may have
-                    # cloud-specific credentials set as env vars (e.g., AWS's
-                    # two env vars). See #1880 and comments above for details.
-                    env=dict(RAY_USAGE_STATS_ENABLED='0'),
-                )
+                subprocess.run([sys.executable, script], check=True, env=env)
 
                 logger.info('Running ray down.')
                 # Stop the workers first to avoid orphan workers.
                 subprocess.run([
                     'ray', 'down', '-y', '--workers-only', self._ray_yaml_path
                 ],
-                               check=True)
+                               check=True,
+                               env=env)
```

Comment:
Any reason we don't use `env=dict()` or `env=dict(RAY_USAGE_STATS_ENABLED='0')` here and the following place as we did before?

Reply:
It will make `ray` not found in PATH. The first `ray up` op is ok as it's technically calling `python <generated>.py` but the latter two ops are directly calling `ray <cmd>`.

Reply:
That makes sense! Can we have a comment here for future reference?

Reply:
Yep, done.